microsoft/fhir-server
# Handle exceptions when schema does not exist

## Description
When Schema is not initialized (SchemaOptions:AutomaticUpdatesEnabled : false), SchemaJob, ExportJob, Task hosting background services throw errors when they try to access stored procedures which do not exists as schema is not initialized. With this PR we are trying to avoid multiple exception loggings. Instead we are logging warnings for the same issues.

## Related issues
Addresses [[81886](https://microsofthealth.visualstudio.com/Health/_workitems/edit/81886)].

## Testing
Manual Testing -
1. Set "SqlServer:SchemaOptions:AutomaticUpdatesEnabled" as false in launchSettings.cs.  
2. Run Fhir server
3. Notice there are no errors are logged in console
4. You should see warning loggings for Could not find stored procedure...

## FHIR Team Checklist
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)
